mana_decay_provider.go

```diff
@@ -244,35 +237,46 @@ func (p *ManaDecayProvider) GenerateManaAndDecayBySlots(amount BaseToken, creati
 		if err != nil {
 			return 0, ierrors.Wrap(err, "failed to decay generated mana in the first epoch")
 		}
+		/*
+			manaGeneratedWholeEpoch, err := p.generateMana(amount, 1<<p.slotsPerEpochExponent)
```

Comment:
what is that here?

Reply:
Don't know, some workings from Olivia... Deleted.